content/en/docs/collector/internal-telemetry.md

```diff
@@ -260,6 +260,53 @@ categorized by instrumentation type.
 | `rpc_server_response_size`        | Measures the size of RPC response messages (uncompressed).                                | Histogram |
 | `rpc_server_responses_per_rpc`    | Measures the number of messages sent per RPC. Should be 1 for all non-streaming RPCs.     | Histogram |
 
+### Telemetry lifecycle
```

Comment:
Hmm, what about clarifying the kind of lifecycle? Also, IMHO it isn't a cycle per se, it's a maturity path. Maybe just drop "lifecycle"?

The official terminology from https://github.com/open-telemetry/oteps/blob/main/text/0232-maturity-of-otel.md#maturity-levels is "maturity levels":

So, how about this:

```suggestion
### Telemetry maturity levels
```

Btw, should this instead be the following instead?

```suggestion
### Signal maturity levels
```

Reply:
I wouldn't want to use Signal here as it may be confused with the signals that the collector receives/processes/exports